packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -329,11 +329,17 @@ export class DocumentDeltaConnection
 	/**
 	 * Submits a new signal to the server
 	 *
-	 * @param message - signal to submit
+	 * @param content - Content of the signal.
+	 * @param targetClientId - When specified, the signal is only sent to the provided client id.
 	 */
-	public submitSignal(message: IDocumentMessage): void {
+	public submitSignal(content: IDocumentMessage, targetClientId?: string): void {
 		this.checkNotDisposed();
-		this.emitMessages("submitSignal", [[message]]);
+
+		if (targetClientId) {
+			throw new Error("Sending signals to specific client ids is not supported");
```

Comment:
```suggestion
			throw new UsageError("Sending signals to specific client ids is not supported");
```

Reply:
actually, i think we should change this to only log an error, as setting the targetid will still send that signal to that client, it will also just send it to all other clients